src/gengowatcher/job_cancellation_manager.py

```
@@ -168,105 +168,147 @@ async def cancel_current_job(self) -> bool:
+                                        else 0,
+                                    }
+                                )
+
+                                # Clear tracking
+                                self.clear_current_job()
+                                self._save_job_state()
+
+                                return True
+                            else:
+                                self.logger.error(
+                                    f"Cancellation may have failed - unexpected response"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)